examples/resources/upcloud_loadbalancer/resource.tf

```diff
@@ -19,4 +19,15 @@ resource "upcloud_loadbalancer" "lb" {
   plan              = "development"
   zone              = var.lb_zone
   network           = resource.upcloud_network.lb_network.id
```

Comment:
This line should probably be removed from the example

Reply:
Yes it should :) It's removed now.